### PR TITLE
#55 library release using sbt-ci-release plugin

### PR DESCRIPTION
**Pull request by lsulak**
Also: updating maintainers and updating sbt

### CODE REVIEW

File: README.md
Comment (lsulak) [reply README.md:392]: Please see the newest changes.

File: README.md
Comment (lsulak) [reply README.md:5]: The short answer is that this Action doesn't yet exist, so the Badge doesn't exist either - it will be there once we merge this PR. Good review!

File: publish.sbt
Comment (lsulak) [reply publish.sbt:51]: I was not sure what the process is around these things since his Absa email should be deactivated by now - but I absolutely agree, I'll redo this change.

File: README.md
Comment (lsulak) [reply README.md:392]: What about the `CODEOWNERS` file, should it be 1:1 to this list of developers, or should it only contain active maintainers?

File: publish.sbt
Comment (lsulak) [reply publish.sbt:57]: What about the `CODEOWNERS` file, should it be 1:1 to this list of developers, or should it only contain active maintainers?

**Review by Zejnilovic — DISMISSED** [review]
Just one Q

File: build.sbt
Comment (lsulak) [reply build.sbt:61]: If I understood it correctly, these three values should be used also for https://github.com/sbt/sbt-header#getting-started, so it might make sense to group them together. So the question is whether we want to put them into `build.sbt` or `publish.sbt`. 

Since this header check is closer to the compilation/validation part, not the publishing part, I think that `build.sbt` is a better candidate - additionally, it would be consistent with our spark-data-standardization: https://github.com/AbsaOSS/spark-data-standardization/blob/master/build.sbt#L63-L66

(btw, our https://github.com/AbsaOSS/sbt-git-hooks/tree/master/.github/workflows doesn't have a licence check - should we add it there & use similar logic?)

Of course, as always, I'm happy to be persuaded, I don't insist on this.

Comment (lsulak) [reply build.sbt]: Changed, based on our discussion elsewhere.

**Review by benedeki — APPROVED** [review]

**Review by Zejnilovic — APPROVED** [review]

**Pull request merged**
